termqt/terminal_widget.py

```diff
@@ -469,7 +469,13 @@ def keyPressEvent(self, event):
 
         if text:
             self.input(text.encode('utf-8'))
-
+    
+    def showEvent(self, event):
```

Comment:
I think `resizeEvent` will be triggered on `show`. Is this not how Qt on Windows works?

Reply:
Hmmm. This is very weird. Maybe a platform-dependent behavior.